src/test/kotlin/com/github/gradle/android/i18n/import/AndroidI18nPluginExtensionTest.kt

```diff
@@ -88,6 +87,15 @@ class AndroidI18nPluginExtensionTest : AbstractUnitTest() {
 
         // Then
         then(exporter).should().export(any(), eq("en"))
-        assertTrue(Paths.get(buildDir.path, "my-i18n-file.xlsx").toFile().exists())
+        assertContains(buildDir, "i18n_", ".xlsx")
+    }
+
+    @Suppress("SameParameterValue")
+    private fun assertContains(buildDir: File?, prefix: String, suffix: String) {
+        val filesMatchingPattern = buildDir?.listFiles { pathname ->
+            val name = pathname.name
+            name.startsWith(prefix) && name.endsWith(suffix)
+        } ?: arrayOf()
+        assertTrue(filesMatchingPattern.isNotEmpty())
```

Comment:
It seems that [AssertJ 3.13](https://assertj.github.io/doc/#assertj-core-3.13.0-isDirectoryContaining) has already implemented this functionality.

The project currently embeds version 3.11 of this API, it could be the opportunity to upgrade this dependency.

WDYT?

Reply:
🤔 
```
java.lang.AssertionError: 
Expecting directory:
  </var/folders/2y/fy2h6xkx5cjfx3t80p7yvrqh0000gq/T/junit2263077166874184705/junit4154659946797015160>
to contain at least one file matching the 'glob:i18n_*.xlsx' pattern but there was none.
The directory content was:
  [i18n_2020-04-15_11-59-27.xlsx]
```

Reply:
Not working using `isDirectoryContaining("glob:pattern")` but seems to work with this:
```
assertThat(buildDir).isDirectoryContaining {
    val pathname = it.name
    pathname.startsWith("i18n_") && pathname.endsWith(".xlsx")
}
```
🤷‍♂️